content/02.body.md

```diff
@@ -36,24 +36,24 @@ How well a network represents the true relationships it attempts to represent de
+For networks with a large inspection bias, reliance on degree can lead to predictions that have good metrics when assessed by cross-validation but little ability to generalize.
 
 Another reason why a reliance on degree can be unfavorable is that degree imbalance can lead to prediction nonspecificity.
 Nonspecific predictions are not made on the basis of the specific connectivity information contained in a network.
-For example, Gillis et al. examined the concept of prediction specificity in the context of gene function prediction and found that many predictions appear to rely primarily on multifunctionality and could be "potentially misleading with respect to causality" [@doi:10.1371/journal.pone.0017258].
+For example, Gillis and Pavlidis examined the concept of prediction specificity in the context of gene function prediction and found that many predictions appear to rely primarily on multifunctionality and could be "potentially misleading with respect to causality" [@doi:10.1371/journal.pone.0017258].
 Degree imbalance leads high-degree nodes to dominate in the predictions made by degree-associated methods [@doi:10.1093/bioinformatics/btv215], which are effective predictors of connections in some biological networks [@doi:10.1186/1752-0509-2-11].
 Consequently, degree-based predictions are more likely nonspecific, meaning the same set of predictions performs well for different tasks.
 
-Depending on the prediction task, edge predictions involving very high degree nodes may be undesired, uninsightful, or nonspecific.
+Depending on the prediction task, edge predictions involving very high-degree nodes may be undesired, uninsightful, or nonspecific.
 While predictions based primarily on degree may be acceptable for some tasks, generating less obvious insights from networks requires drawing inferences from the specific connections and network structure between nodes.
 Model evaluation is challenging in this context: nonspecific or trivial predictions can dominate performance evaluations and may actually be correct, even if they are not the desired outputs of the predictive model.
 For example, predicting that the highest degree node in a network shares edges with the remaining nodes to which it is not connected will often lead to many correct predictions, despite this prediction being generic to all other nodes in the network.
 
 Degree is important in edge prediction, but it can cause undesired effects.
-Degree-based features should often be included in the interpretation of predictions to disentangle desired from non-desired effects and to effectively evaluate and compare predictive models.
+Degree-based features should often be included in the interpretation of predictions to disentangle desired from undesired effects and to effectively evaluate and compare predictive models.
```

Comment:
we might want to make this change (nondesired --> undesired) in the Gigascience version as well